client/src/components/NavBar/AuthBars/LoggedInBar.tsx

```diff
@@ -4,30 +4,40 @@ import Typography from '@material-ui/core/Typography';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
 import AuthMenu from '../../AuthMenu/AuthMenu';
+import { useLocation } from 'react-router-dom';
 
 const LoggedInBar = (): JSX.Element => {
   const classes = useStyles();
+  const { pathname } = useLocation();
 
   return (
     <Grid container className={classes.navButtons}>
       <Grid item>
         <Button component={Link} to="/notifications" color="secondary" size="large" variant="text">
-          <Typography variant="h3">Notifications</Typography>
+          <Typography className={pathname === '/' ? classes.homeTypography : classes.typography} variant="h3">
```

Comment:
Optional: You can use a lightweight package list [clsx](https://www.npmjs.com/package/clsx) to conditionally apply class names - makes things a lot cleaner